src/syscall/types_windows_arm64.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package syscall

```

Comment:
Is this intentionally duplicated from src/cmd/vendor/golang.org/x/sys/windows/types_windows_arm64.go ?

Reply:
Considering arm32 does the same, yes this is intentional.